src/include/ipc/info.h

```diff
@@ -33,6 +33,7 @@
 #define SOF_IPC_INFO_LOCKS		BIT(1)
 #define SOF_IPC_INFO_LOCKSV		BIT(2)
 #define SOF_IPC_INFO_GDB		BIT(3)
+#define SOF_IPC_INFO_IMR_RESTORE	BIT(4)
```

Comment:
ABI?

Reply:
What about ```SOF_IPC_INFO_PM_D3_PERSISTENT``` to mean no need to re-copy FW to DSP after PM D3.

Reply:
Probably does not need ABI bump since the structures have not changed, we are just adding a new value here.

Reply:
> What about `SOF_IPC_INFO_PM_D3_PERSISTENT` to mean no need to re-copy FW to DSP after PM D3.

To avoid too long naming, `SOF_IPC_INFO_D3_PERSISTENT` sounds good to me.